Makefile

```diff
@@ -315,7 +315,7 @@ endif
 
 .PHONY: docs-crates
 docs-crates: musl $(GEN_TS) $(GEN_RS) ## Build the crate (documentation)
-	$(cargo) doc --target $(uname_m)-unknown-linux-musl --no-deps --package auraed
```

Comment:
Not sure if this breaks anything. Assuming our CI would run with the correct rust target by default, I think we're fine. This change allows all the rustdocs to be unified and have links to each crate on the left navigation.

![image](https://github.com/aurae-runtime/aurae/assets/16668493/4704fe38-ef4e-4c28-bd0f-39f78fafcddc)

Reply:
> Not sure if this breaks anything. Assuming our CI would run with the correct rust target by default, I think we're fine. This change allows all the rustdocs to be unified and have links to each crate on the left navigation.
> 
> ![image](https://private-user-images.githubusercontent.com/16668493/296555320-4704fe38-ef4e-4c28-bd0f-39f78fafcddc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUyNDIyNjAsIm5iZiI6MTcwNTI0MTk2MCwicGF0aCI6Ii8xNjY2ODQ5My8yOTY1NTUzMjAtNDcwNGZlMzgtZWY0ZS00YzI4LWJkMGYtMzlmNzhmYWZjZGRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE0VDE0MTkyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwZTYwMDgwODk1MTczMWU2Mzc4NmZjOTNkM2VjNWFjMjAzNmFjNTcwYjM1YjQ2NDU4ZThjNjdiMmQ0NzcyMDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sdvvxQ_YlswKUUHIdwkJfKNQx2q09FXuQKw8CSaDSfo)

it shouldn't, no. our ci doesn't use the right target (musl) but for docs I suspect it doesnt matter...